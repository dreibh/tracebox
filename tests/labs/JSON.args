-q -p IP/TCP -j
