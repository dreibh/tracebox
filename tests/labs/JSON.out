{ "addr": "1.2.3.4", "name": "1.2.3.4", "max_hops": 64, "Hops": [ { "hop": 1, "from": "1.1.1.1", "Modifications": [ "IP::TTL", "IP::Protocol" ], "Aditions": [ "RawLayer" ], "Deletions": [ "TCP" ] }, { "hop": 2, "from": "1.2.3.4", "Modifications": [ "IP::TTL", "IP::Protocol", "IP::CheckSum" ], "Aditions": [ "RawLayer" ], "Deletions": [ "TCP" ] } ] }
